lib/tasks/travel_advice/tmp_swap_travel_advice_link.rake

```diff
@@ -15,15 +15,20 @@ namespace :travel_advice do
       links = Services.publishing_api.get_links(edition["content_id"])["links"]
       next unless links && links["ordered_related_items"]
 
-      index_of_link = links["ordered_related_items"].index(take_out)
+      begin
+        index_of_link = links["ordered_related_items"].index(take_out)
 
-      links["ordered_related_items"][index_of_link] = swap_in if index_of_link
+        links["ordered_related_items"][index_of_link] = swap_in if index_of_link
 
-      Services.publishing_api.patch_links(
-        edition["content_id"],
-        links: links,
-        bulk_publishing: true,
-      )
+        Services.publishing_api.patch_links(
+          edition["content_id"],
+          links: links,
+          bulk_publishing: true,
+        )
+      rescue StandardError
```

Comment:
Should this be more specific? i.e. catching the GdsApi error rather than everything?

Reply:
Oh yeah - thanks